notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data = CLD.handle_missing_codes_encoded_features(data, prefixes, missing_codes, encoded_columns_names_dict)
</pre><p>I haven't seen the functions file yet, but why do you send the&nbsp;<em>prefixes&nbsp;</em>list if you have the&nbsp;<em>encoded_columns_names_dict</em>&nbsp;dictionary with the prefixes as the index?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>